test/frontend.js

```diff
@@ -15,7 +15,10 @@ const exec = require('child_process').execFileSync
 const fs = require('fs')
 
 describe('Frontend', function () {
-  var n = Nightmare({ show: !!process.env.SHOW_BROWSER, height: 900 })
+  const n = Nightmare({
+    show: !!process.env.SHOW_BROWSER,
```

Comment:
If you make this a `true` value you can watch the test cases run in an electron window